Public/Sdk/Experimental/NodeJs/Yarn/yarn.dsc

```diff
@@ -103,6 +101,9 @@ export function install(args: Arguments) : Result {
                         d`${Context.getMount("AppData").path}/Microsoft/SystemCertificates/My/Keys`, // Cache for certificats
                         d`${Context.getMount("AppData").path}/Microsoft/VisualStudio Services/7.0/Cache`, // Cache for visaul studio services
                     ],
+                    passThroughEnvironmentVariables: [
+                        "NUGET_CREDENTIALPROVIDERS_PATH"

```

Comment:
Why not make this a command line set GlobalPassthroughEnvVar to match the scope being untracked via the command line?

Reply:
For the ones that needed both to be in CB and Dev build, I think it makes more sense to put in in the spec instead of putting it in both CB and Dev command line.

Reply:
changed